File: Lecture-8/MultiplyBinary.java
import java.util.Scanner;

public class MultiplyBinary {
    public static void main(String[] args) {
        try (Scanner sc = new Scanner(System.in)) {

            System.out.print("Enter 1st binary number: ");
            String binary1 = sc.next();
            System.out.print("Enter 2nd binary number: ");
            String binary2 = sc.next();

            int num1 = Integer.parseInt(binary1, 2);
            int num2 = Integer.parseInt(binary2, 2);

            int multiply = num1 * num2;
            String result = Integer.toBinaryString(multiply);

            System.out.print(binary1 + " x " + binary2 + " = " + result);
        }
    }
}

File: DS-Algo-with-Python/Lecture-8/MultiplyBinary.py
binary1 = input("Enter 1st binary number: ")
binary2 = input("Enter 2nd binary number: ")

num1 = int(binary1, 2)
num2 = int(binary2, 2)
multiply = num1 * num2

result = bin(multiply).replace("0b", "")
print(f"{binary1} x {binary2} = {result}")
